packages/workflow/src/internals.ts

```diff
@@ -741,7 +774,9 @@ export class Activator implements ActivationHandler {
       args: arrayFromPayloads(this.payloadConverter, activation.input),
       signalName,
       headers: headers ?? {},
-    }).catch(this.handleWorkflowFailure.bind(this));
+    })
+      .catch(this.handleWorkflowFailure.bind(this))
```

Comment:
We need to `untrackPromise()` both the promise resulting from the call on `.catch()`, and the one from `.finally()`.